x/crosschain/keeper/cctx_orchestrator_validate_outbound.go

```diff
@@ -53,7 +53,9 @@ func (k Keeper) ValidateOutboundZEVM(
 			cctx.InboundParams.Amount,
 		)
 		if err != nil {
-			cctx.SetAbort(fmt.Sprintf("%s : %s", depositErr, err.Error()))
+			cctx.SetAbort(
+				"Aborted because an error was raised while trying to revert cctx",
+				fmt.Sprintf("Deposit error: %s, Processing error: %s", depositErr, err.Error()))
```

Comment:
Can we declare `Deposit Error` and `Processing Error` ( the strings ) as constants and then use them when ever setting the error message 

Also, Something more generic, like `Outbound Error` and `Revert Outbound Error`, would be helpful.

Reply:
We just have a couple logs using deposit error, so I'll manually add them.